js/packages/candy-machine-ui/.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_CANDY_MACHINE_ID=<YOUR CANDY MACHINE PROGRAM ID>
+
+REACT_APP_SOLANA_NETWORK=devnet
+REACT_APP_SOLANA_RPC_HOST=https://api.devnet.solana.com
```

Comment:
what do you think about renaming this file to `.env.example` and adding `js/packages/candy-machine-ui/.env` path to `.gitignore`? Benefit: when people clone this repo and customize their own `.env` settings, git ignores the change

Reply:
Good idea!